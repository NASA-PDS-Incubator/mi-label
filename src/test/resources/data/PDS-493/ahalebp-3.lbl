PDS_VERSION_ID                     = PDS3                                     
                                                                              
RECORD_TYPE                        = "FIXED_LENGTH"                           
RECORD_BYTES                       = 2880                                     
FILE_RECORDS                       = 3155                                     
                                                                              
^HEADER                            = "ahalebp.fit"                          
^IMAGE                             = ("ahalebp.fit", 3)                       
                                                                              
DATA_SET_ID                        = "MSX-C-SPIRIT3-3-MSXSB-V1.0"             
PRODUCT_NAME                       = "AHALEBP"                                
PRODUCT_ID                         = "HALEBOPP_AHALEBP_FIT"                   
INSTRUMENT_HOST_ID                 = "MSX"                                    
INSTRUMENT_HOST_NAME               = "MIDCOURSE SPACE EXPERIMENT"             
INSTRUMENT_ID                      = "SPIRIT3"                                
INSTRUMENT_NAME                    = "SPATIAL INFRARED IMAGING TELESCOPE"     
TARGET_NAME                        = "C/HALE-BOPP (1995 O1)"                  
TARGET_TYPE                        = "COMET"
/*                                                                            
 Start Time                                                                
 ============= */                                    
START_TIME                         = 1996-10-08T01:24:25.660                  
STOP_TIME                          = 1996-10-08T01:53:40.660                  
PRODUCT_CREATION_TIME              = 2005-12-13  /* File uploaded to OLAF */  
CITATION_DESC                      = "Kraemer, K., Lisse, C.M., Price, S.,    
        Mizuno, D., Walker, R.G., Farnham, T.L., and Makinen, T., AHALEBP.    
        MSX-C-SPIRIT3-3-MSXSB-V1.0:HALEBOPP_AHALEBP_FIT. NASA Planetary Data  
        System, 2005."                                                        
REFERENCE_KEY_ID                   = "KRAEMERETAL2005"                        
RIGHT_ASCENSION                    = 262.7000122 <DEGREES>                    
DECLINATION                        = -4.800000191 <DEGREES>                   
FILTER_NAME                        = "A"                                      
CENTER_FILTER_WAVELENGTH           = .00000828 <METERS>                       
EXPOSURE_DURATION                  = 1755 <SECONDS>                           
                                                                              
OBJECT                             = HEADER                                   
 BYTES                             = 5760                                     
 HEADER_TYPE                       = "FITS"                                   
 INTERCHANGE_FORMAT                = "BINARY"                                 
 RECORDS                           = 2                                        
END_OBJECT                         = HEADER                                   
                                                                              
                                                                              
OBJECT                             = IMAGE                                    
  LINES                            = 2420                                     
  LINE_SAMPLES                     = 938                                      
  SAMPLE_BITS                      = 32                                       
  SAMPLE_TYPE                      = "IEEE_REAL"                              
  DERIVED_MINIMUM                  = -5.7450543E-5                            
  DERIVED_MAXIMUM                  = 3.056716E-4                              
  AXIS_ORDER_TYPE                  = "FIRST_INDEX_FASTEST"                    
  LINE_DISPLAY_DIRECTION           = "UP"                                     
  SAMPLE_DISPLAY_DIRECTION         = "RIGHT"                                  
  UNIT                             = "WATTS/METER**2-STERADIAN"               
  HORIZONTAL_PIXEL_FOV             = 18.3 <ARCSECONDS>                        
  VERTICAL_PIXEL_FOV               = 18.3 <ARCSECONDS>                        
  CELESTIAL_NORTH_CLOCK_ANGLE      = 42.0 <DEGREES>                           
  DESCRIPTION                      = "MSX Spirit III infrared image of small  
        solar system bodies"                                                  
END_OBJECT                         = IMAGE                                    
                                                                              
END                                                                           
