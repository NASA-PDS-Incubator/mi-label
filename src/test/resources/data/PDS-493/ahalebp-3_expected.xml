<?xml version="1.0" encoding="UTF-8" standalone="no"?>

<?xml-model href="https://pds.nasa.gov/pds4/pds/v1/PDS4_PDS_1A10.sch"
  schematypens="http://purl.oclc.org/dsdl/schematron"?>

<Product_Observational xmlns="http://pds.nasa.gov/pds4/pds/v1" 
    xmlns:pds="http://pds.nasa.gov/pds4/pds/v1" 
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
    xsi:schemaLocation="http://pds.nasa.gov/pds4/pds/v1 https://pds.nasa.gov/pds4/pds/v1/PDS4_PDS_1A10.xsd">
  <Identification_Area>
    <logical_identifier>urn:nasa:pds:data:msx-c-spirit3-3-msxsb-v1.0:halebopp_ahalebp_fit</logical_identifier>
    <version_id>1.0</version_id>
    <title>ahalebp</title>
    <!-- hardcoding IM version for test case. will test this elsewhere -->
    <information_model_version>1.10.1.0</information_model_version>
    <product_class>Product_Observational</product_class>
    <Modification_History>
      <Modification_Detail>
        <!-- hardcoding modification date for test case. will test this elsewhere -->
        <modification_date>2018-07-26</modification_date>
        <version_id>1.0</version_id>
        <description>Auto generated PDS4 product label created by the Transform Tool.</description>
      </Modification_Detail>
    </Modification_History>
  </Identification_Area>
  <Observation_Area>
    <Time_Coordinates>
      <start_date_time>1996-10-08T01:24:25.660Z</start_date_time>
      <stop_date_time>1996-10-08T01:53:40.660Z</stop_date_time>
    </Time_Coordinates>
    <Investigation_Area/>
    <Observing_System>
      <Observing_System_Component>
        <name>midcourse space experiment</name>
        <type>Spacecraft</type>
      </Observing_System_Component>
      <Observing_System_Component>
        <name>spatial infrared imaging telescope</name>
        <type>Instrument</type>
      </Observing_System_Component>
    </Observing_System>
    <Target_Identification>
      <name>c/hale-bopp (1995 o1)</name>
      <type>Planet</type>
    </Target_Identification>
  </Observation_Area>
  <File_Area_Observational>
    <File>
      <file_name>ahalebp.fit</file_name>
    </File>
    <Array_2D_Image>
      <offset unit="byte">5760</offset>
      <axes>2</axes>
      <axis_index_order>Last Index Fastest</axis_index_order>
      <Element_Array/>
      <Axis_Array>
        <axis_name>Line</axis_name>
        <elements>2420</elements>
        <sequence_number>1</sequence_number>
      </Axis_Array>
      <Axis_Array>
        <axis_name>Sample</axis_name>
        <elements>938</elements>
        <sequence_number>2</sequence_number>
      </Axis_Array>
    </Array_2D_Image>
  </File_Area_Observational>
</Product_Observational>
